hnn_core/feed.py

```diff
@@ -69,14 +163,16 @@ def feed_event_times(feed_type, target_cell_type, params, gid):
         Parameters of the external input feed, arranged into a dictionary.
     gid : int
         The cell ID.
+    trial_idx : int
+        The index number of the current trial of a simulation.
 
     Returns
     -------
     event_times : list
         The event times at which spikes occur.
     """
     prng, prng2 = _get_prng(
-        seed=params['prng_seedcore'],
+        seed=params['prng_seedcore'] + trial_idx,
         gid=gid,
```

Comment:
separate PR and let's make an issue. But the example got me thinking that `prng` shouldn't depend on gid. Basically use `prng` to create 100 random numbers to function as the "child" `seeds`. That way it's deterministic no matter what order you add the feeds

Reply:
+1 for separate PR _and_ getting rid of gid-dependency. I don't see how the order won't still matter, though?

Reply:
Humm ... maybe I was confused. So my thinking is basically something like:

https://github.com/jonescompneurolab/hnn-core/blob/8171482ed7bc4a90a52c063057acd298117b3ef8/hnn_core/feed.py#L38

would become:

```py
seed = seed + target_cell_gid
```

or alternatively:

```py
seeds = rng.randint(seedcore)
seed = seeds[target_cell_gid]
```
and if real cells always come before artificial cells, then there is no risk of the seed changing? Not sure if this is clearer but we can discuss IRL